src/app/views/cheat-sheets/belt-throughput/belt-throughput.component.html

```diff
@@ -3,10 +3,10 @@
         <strong><a href="https://wiki.factorio.com/Transport_belt" target="_blank" rel="noopener">Transport Belt</a> Throughput</strong> - Items per second (i/s) passed over one belt tile.
     </p>
     <p class="card-text">
-        <strong>Transport Belt Density</strong> - Maximum Items fit in one tile. (<strong>8 items</strong> for all belts)
```

Comment:
This is a definition though
maybe do 
Maximum Items that fit in one tile: 8 items for all belts
?